### PR TITLE
Update .roll to use new dice emojis

### LINKED ISSUE #440: .dice doesn’t display emojis
## Description
After an update to the server's dice emojis' names (together with updated designs), the output of `.dice` will be the string of the old emoji names, instead of the new ones. This causes it to be displayed as text.

## Steps to Reproduce

Send `.roll` in any channel that accepts the command.

## Expected Behaviour

Assuming that the output is 3, the returned string should be:
`:dice_3:`

This would be converted to the correct emoji by Discord.

## Actual Behaviour

The output of `.dice` will be the string of the old emoji names, like this:
`:terning3:`

This is not displayed to an emoji.

## Known Impacted Platforms

- [x] Web
- [x] Desktop
- [x] Android App
- [x] iOS App

## Possible Solutions

Change the emoji names in the command to the new format:

- `:dice_1:`
- `:dice_2:`
- ...
- `:dice_6:`

## Would you like to implement a fix?

***Note: For high-priority or critical bugs, fixes may be implemented by staff.***

- [x] I'd like to implement the bug fix
- [ ] Anyone can implement the bug fix

### PR DESCRIPTION
**Pull request by gustavwilliam**
## Relevant Issues

Closes #440.

## Description

- Emojis in `constants.py` were replaced with the new emojis, following the dice redesign
- `.roll` command was updated to use the new emoji names in `constants.py`

## Did you:

- [x] Join the [**Python Discord Community**](https://discord.gg/python)?
- [x] If dependencies have been added or updated, run `pipenv lock`?
- [x] **Lint your code** (`pipenv run lint`)?
- [x] Set the PR to **allow edits from contributors**?

### CODE REVIEW

File: bot/exts/evergreen/fun.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ async def roll(self, ctx: Context, num_rolls: int = 1) -> None:
         elif num_rolls < 1:
             output = ":no_entry: You must roll at least once."
         for _ in range(num_rolls):
-            terning = f"terning{random.randint(1, 6)}"
+            terning = f"dice_{random.randint(1, 6)}"
```
Comment (Den4200) [inline bot/exts/evergreen/fun.py:44]: Maybe also rename the variable to `dice` or `die`?

**Review by Den4200 — CHANGES_REQUESTED** [review]
Nice, only one small request.

File: bot/exts/evergreen/fun.py (hunk 1 above)
Comment (gustavwilliam) [reply bot/exts/evergreen/fun.py:44]: Sure; we could do that. The emojis on the server are called `dice_X`, though.

If we rename this, I think we should rename the server icons as well.

**Review by Den4200 — APPROVED** [review]
Looks good to me, thanks!

**Review by kwzrd — APPROVED** [review]
Thanks for the fix!

**Pull request merged**